CHANGES.md

```diff
@@ -42,6 +42,7 @@ Apollo 2.1.0
 * [Portal-UI adds serverConfig configuration management of ApolloConfigDB](https://github.com/apolloconfig/apollo/pull/4680)
 * [Enable login authentication for eureka](https://github.com/apolloconfig/apollo/pull/4663)
 * [Add github action to publish docker image](https://github.com/apolloconfig/apollo/pull/4685)
+* [Fix apolloconfigdb.sql apolloportaldb.sql AppId length settings are inconsistent](https://github.com/apolloconfig/apollo/pull/4725)
```

Comment:
As #4721 is merged, please add the changes to the new CHANGES.md file.

Reply:
OK, I have updated the changes to the new CHANGES.md file.